pyvista/jupyter/notebook.py

```diff
@@ -60,6 +64,52 @@ def handle_plotter(
     return show_static_image(plotter, screenshot)
 
 
+def show_vtk_wasm(plotter: Plotter, **kwargs) -> IFrame:  # noqa: ARG001  # numpydoc ignore=RT01
+    """Display a VTK WebAssembly widget in a jupyter notebook."""
+    try:
+        from IPython.display import IFrame  # noqa: PLC0415
+        import trame_vtklocal  # noqa: F401, PLC0415
+        import vtk  # noqa: F401, PLC0415
+    except ImportError as e:
+        msg = (
+            'VTK-WASM backend requires VTK>=9.3 and trame-vtklocal. '
+            'Install: pip install --extra-index-url https://wheels.vtk.org '
+            '"vtk>=9.3" trame-vtklocal'
```

Comment:
```suggestion
```suggestion
                'VTK-WASM backend requires VTK>=9.4 and trame-vtklocal. Install with:'
                '    pip install --extra-index-url vtk>=9.4 trame-vtklocal'
```
```